libs/coin-framework/src/bot/types.ts

```diff
@@ -120,8 +120,6 @@ export type AppSpec<T extends TransactionCommon> = {
   scanAccountsRetries?: number;
   // if define, will run the mutations {multipleRuns} times in order to cover 2 txs in the same run and detect possible issues at the "second tx time"
   multipleRuns?: number;
-  // define the frequency of exporting/importing back the account to simulate mobile export
-  crossAccountFrequency?: number;
```

Comment:
I have removed in the bot a feature that was problematic to keep: the ability to test that an account still works after "crossing" it.

since the logic of cross.ts was moved to live-wallet, and at the beginning of the rework LLC was not even depending on live-wallet, i initially did this move (but eventually it start depending on)

if you think this is important to keep in the bot, i can keep it, but the mecanism of `cross.ts` is something we will eventually sunset (wallet sync to replace it) so => i can always move it back, but we will eventually sunset it anyway.